packages/eth2.0-state-transition/src/constants/network.ts

```diff
@@ -0,0 +1,28 @@
+// gossip
+
+export const BLOCK_TOPIC = "/eth2/beacon_block";
+export const ATTESTATION_TOPIC = "/eth2/beacon_attestation";
+export const SHARD_ATTESTATION_TOPIC = "/eth2/shard{shard}_attestation";
+export const SHARD_SUBNET_COUNT = 10;
+
+// req/resp
+
+export type RequestId = string;
+
+export enum Method {
+  Hello = "hello",
+  Goodbye = "goodbye",
+  BeaconBlocksByRange = "beacon_blocks_by_range",
+  BeaconBlocksByRoot = "beacon_blocks_by_root",
+}
+
+export enum Encoding {
+  ssz = "ssz",
+}
+
+export const ERR_INVALID_REQ = "invalid request";
+export const ERR_RESP_TIMEOUT = "response timeout";
+export const REQ_RESP_MAX_SIZE = 2 ** 22; // ~4MB
+export const TTFB_TIMEOUT = 5 * 1000; // 10 sec
+export const RESP_TIMEOUT = 10 * 1000; // 10 sec
+
```

Comment:
Woah this file should be in networking module though

Reply:
sorry, this was from copy and pasting the utils folder from `lodestar`. just removed it.